spg/Animal/lib/RaceHorse.pm

```diff
@@ -77,34 +83,24 @@ sub new {
 
   @{$self}{@stats} = @{$stats_data}{@stats};
 
-  $self;
-}
-
-sub won { $_[0]->{wins}++; }
-
-sub placed { $_[0]->{places}++; }
-
-sub showed { $_[0]->{shows}++; }
-
-sub lost { $_[0]->{losses}++; }
-
-sub standings {
-  my ( $self ) = @_;
-  join ', ', map "$self->{$_} $_", @stats;
+  return $self;
 }

```

Comment:
Moose also has its own constructor via `new` ([relavant doc](http://search.cpan.org/dist/Moose/lib/Moose/Manual/Construction.pod#OBJECT_CONSTRUCTION_HOOKS)) - this should use `BUILD`.